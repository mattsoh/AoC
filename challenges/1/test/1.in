13413413434635756444