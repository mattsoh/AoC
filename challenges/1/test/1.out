13413413434635756445